enterprise/frontend/src/metabase-enterprise/audit_app/components/AuditParameters.jsx

```diff
@@ -48,8 +49,12 @@ export default class AuditParameters extends React.Component {
   }, DEBOUNCE_PERIOD);
 
   render() {
-    const { parameters, children, buttons } = this.props;
+    const { parameters, children, buttons, isBlank } = this.props;
     const { inputValues, committedValues } = this.state;
+
+    const disabled =
```

Comment:
If we did not `location.reload();` in `ErrorOverview.jsx` this would lead to a bug when users entered something in inputs, then removed and rerun questions so that they all become correct and the result is empty. So it's better to check that Object.values(inputValues) has only empty strings

I'm wondering if we can pass reload from `QuestionLoadAndDisplay` to call it inside `ErrorOverview.jsx` after we rerun queries.

And here wdyt about renaming `isBlank` to `hasResults` because `isBlank` can be thought of as inputs blank or something like that at first when reading

Reply:
Yeah, I've gone through 3 names for `isBlank` by now. `hasResults` doesn't sound too bad

I don't think the reload from `QuestionLoadAndDisplay` actually triggers the card itself? Unless you want to separately have another instance that calls the card. We can probably do it, in either case

Checking `Objecgt.values` for empty strings sgtm, willdo

Reply:
> Yeah, I've gone through 3 names for `isBlank` by now. `hasResults` doesn't sound too bad
> 
> I don't think the reload from `QuestionLoadAndDisplay` actually triggers the card itself? Unless you want to separately have another instance that calls the card. We can probably do it, in either case
> 
> Checking `Object.values` for empty strings sgtm, willdo

Reply:
I created a PR with an example to explain what I meant, please take a look: https://github.com/metabase/metabase/pull/18092

Reply:
I didn't know that the `ref` reload trick was still kosher, I saw it in other places in the codebase and thought it was deprecated. I can stick this bit in, sure

Reply:
It is not elegant, but we don't have a better way do to this currently